src/lib/chains/tron/types.ts

```diff
@@ -0,0 +1,15 @@
+import { TokenConfig } from "../evm";
+
+export type TronConfig = {
```

Comment:
Can we get a comment/documentation on the fields of this type?  It's not clear what a fullNode/solidityNode/eventServer is without someone digging into Tron documents?

We will need to do a full documentation sweep on the whole SDK, but documenting this now will help.